data/analytics/events.yml

```diff
@@ -303,14 +303,103 @@
         - name: method
         - name: link_domain
 
-- name: call out box
+- name: call out boxes
   events:
     - name: call out box link click
-      implemented: false
-      priority: medium
+      description: Link tracking for call to action, warning text, advisory callout, information callout, and content banner components.
+      tracker: link_tracker
+      implemented: true
+      priority: low
+      data:
+      - name: event
+        value: event_data
+      - name: event_data
+        value:
+          - name: event_name
+            value: navigation
+          - name: type
+            value: callout
+          - name: url
+          - name: text
+          - name: external
+          - name: method
+          - name: link_domain
+
+- name: contact
+  events:
+    - name: form response
+      description: 'When the user clicks "Send message" on the contact form.'
+      tracker: form_tracker
+      examples:
+        - text: Contact GOV.UK
+          link: https://www.gov.uk/contact/govuk
+      implemented: true
+      priority: low
+      data:
+      - name: event
+        value: event_data
+      - name: event_data
+        value:
+          - name: event_name
+            value: form_response
+          - name: type
+            value: contact
+          - name: action
+            value: Send message
+          - name: section
+            value: "What's it to do with?"
+          - name: text
+            value: "The chosen 'What's it to do with?' radio button text, and the URL of the page theyre contacting us about, if relevant."
+          - name: tool_name
+            value: Contact GOV.UK
+    - name: form error
+      description: When the server's form validation fails, and returns the user to the contact form with errors.
+      tracker: auto_tracker
+      examples:
+        - text: Contact GOV.UK
+          link: https://www.gov.uk/contact/govuk
+      implemented: true
+      priority: low
+      data:
+      - name: event
+        value: event_data
+      - name: event_data
+        value:
+          - name: event_name
+            value: form_error
+          - name: type
+            value: contact
+          - name: section
+            value: "(Comma separated list of form inputs that errored)"
```

Comment:
Is using brackets like this a convention we use? Also wondering if it would be worth writing an example, so this would become something like:

`Comma separated list of form inputs that errored, e.g. "name,message"`